cluster/kubectl.sh

```diff
@@ -121,12 +112,15 @@ elif [[ "${KUBERNETES_PROVIDER}" == "gke" ]]; then
     "--zone=${ZONE}"
     "--cluster=${CLUSTER_NAME}"
   )
+elif [[ "$KUBERNETES_PROVIDER" == "vagrant" ]]; then
+  # When we are using vagrant it has hard coded auth.  We repeat that here so that
+  # we don't clobber auth that might be used for a publicly facing cluster.
+  config=(
+    "--auth-path=$HOME/.kubernetes_vagrant_auth"
+  )
 fi
 
-detect-master > /dev/null
-if [[ -n "${KUBE_MASTER_IP-}" && -z "${KUBERNETES_MASTER-}" ]]; then
-  export KUBERNETES_MASTER=https://${KUBE_MASTER_IP}
-fi
+echo "current-context: \"$(${kubectl} config view -o template --template='{{index . "current-context"}}')\""
```

Comment:
The output is an empty string if the user has no current-context. Still,
```
current-context: 
```
is potentially ambiguous output, so changed to enclose the current-context in quotes. Output is now
```
// if user has current-context=my-context
current-context: "my-context"

// if user has empty current-context
current-context: ""
```